Tests/FluentAssertions.Specs/Formatting/EnumerableExtensionsSpecs.cs

```diff
@@ -1,34 +1,31 @@
 ﻿using System.Collections.Generic;
-
 using FluentAssertions.Formatting;
-
 using Xunit;
 
-namespace FluentAssertions.Specs
+namespace FluentAssertions.Specs.Formatting;
```

Comment:
Changed namespace from `FluentAssertions.Specs` to `FluentAssertions.Specs.Formatting`.